src/safe_cli/safe_creator.py

```diff
@@ -88,8 +105,8 @@ def setup_argument_parser():
 
 def main(*args, **kwargs) -> EthereumTxSent:
     parser = setup_argument_parser()
-    print_formatted_text(text2art("Safe Creator"))  # Print fancy text
     args = parser.parse_args()
+    print_formatted_text(text2art("Safe Creator"))  # Print fancy text
```

Comment:
This is moved so that the logo is not displayed when the --help and --version commands are executed.